src/utils.test.js

```diff
@@ -0,0 +1,56 @@
+import { expect, it, describe } from "vitest";
+import { newDate, chomp, mapChomp, Classifier } from "./utils.js";
+
+describe("newDate", () => {
+  it("parses DD/MM/YYYY dates correctly", () => {
+    let d = newDate("10/12/1999");
+    expect(d).toEqual(new Date(1999, 12, 10));
+  });
+  ["10/12/19", "10/12/1999/2", "12/14/1999"].map((d) =>
+    it(`errors on invalide date: ${d}`, () => {
+      let D = newDate(d);
+      expect(D).toBe(null);
+    }),
+  );
+});
+
+describe("chomp", () => {
+  it("should remove all trailing and preceding space", () => {
+    expect(chomp(" lol")).toBe("lol");
+    expect(chomp("lol ")).toBe("lol");
+  });
+});
+
+describe("mapChomp", () => {
+    it("should remove all trailing and preciding spaces on elements and return them in an array", () => {
+        const r = mapChomp(" this ; is;a;  cruel cruel; world");
+        expect(r[0]).toBe("this");
+        expect(r[1]).toBe("is");
+        expect(r[2]).toBe("a");
+        expect(r[3]).toBe("cruel cruel");
+        expect(r[4]).toBe("world");
+    })
+})
+
+describe("Classifier", () => {
+    it("should return an object of type Classifier", () => {
```

Comment:
In my opinion this test does nothing. You're testing the `new` operator. I'd remove this.